### ISSUE TITLE
proper syntax highlighting in bazelrc

### ISSUE THREAD

**Issue by nkoroste** [bazelbuild/intellij#6584]
### Description of the feature request:

The title says it all

**Comment by iancha1992** [comment]
Can you please give us an example? What do you mean? @nkoroste

**Comment by tpasternak** [comment]
@iancha1992 Just a syntax highlighting and code completion for `.bazelrc` file syntax. Unfortunately we don't have capacity to work on that.

**Comment by mtoader** [comment]
@tpasternak I have a working language bit (with highlighting) targeting hirschgarten. Would people be open to incorporate in there?

**Comment by tpasternak** [comment]
Sure, please just open a PR

**Comment by agluszak** [comment]
@mtoader yes, we are very much open to contributions! If you have any questions please ping me :)

**Comment by tpasternak** [comment]
https://github.com/JetBrains/hirschgarten/pull/56

**Comment by tpasternak** [comment]
@nkoroste did you have a chance to try the new (bsp) plugin? the bazelrc support is already there. If all is good in the new plugin, then I probably won't focus on backporting it here, as I simply don't have capacity do do it :(

**Comment by mtoader** [comment]
I think it's fine to close this. The main work is happening in the new bsp plugin.
